cloud/user/user.go

```diff
@@ -55,6 +62,53 @@ func CreateInvite(email, role string, out io.Writer, client astrocore.CoreClient
 	return nil
 }
 
+func UpdateUserRole(email, role string, out io.Writer, client astrocore.CoreClient) error {
+	var userID string
+	err := IsRoleValid(role)
+	if err != nil {
+		return err
+	}
+	ctx, err := context.GetCurrentContext()
+	if err != nil {
+		return err
+	}
+	if ctx.OrganizationShortName == "" {
+		return ErrNoShortName
+	}
+	// Get all org users. Setting limit to 1000 for now
```

Comment:
Is 1000 users too much? What order are they coming in?

Reply:
I think they are coming in a random order right now. Not sure what order they should be coming in